keyboards/ghost54/info.json

```diff
@@ -0,0 +1,126 @@
+                {"matrix": [0, 1], "x": 1, "y": 0},
+                {"matrix": [0, 2], "x": 2, "y": 0},
+                {"matrix": [0, 3], "x": 3, "y": 0},
+                {"matrix": [0, 4], "x": 4, "y": 0},
+                {"matrix": [0, 5], "x": 5, "y": 0},
+
+                {"matrix": [1, 0], "x": 0, "y": 1},
+                {"matrix": [1, 1], "x": 1, "y": 1},
+                {"matrix": [1, 2], "x": 2, "y": 1},
+                {"matrix": [1, 3], "x": 3, "y": 1},
+                {"matrix": [1, 4], "x": 4, "y": 1},
+                {"matrix": [1, 5], "x": 5, "y": 1},
+
+                {"matrix": [2, 0], "x": 0, "y": 2},
+                {"matrix": [2, 1], "x": 1, "y": 2},
+                {"matrix": [2, 2], "x": 2, "y": 2},
+                {"matrix": [2, 3], "x": 3, "y": 2},
+                {"matrix": [2, 4], "x": 4, "y": 2},
+                {"matrix": [2, 5], "x": 5, "y": 2},
+
+                {"matrix": [3, 0], "x": 0, "y": 3},
+                {"matrix": [3, 1], "x": 1, "y": 3},
+                {"matrix": [3, 2], "x": 2, "y": 3},
+                {"matrix": [3, 3], "x": 3, "y": 3},
+                {"matrix": [3, 4], "x": 4, "y": 3},
+                {"matrix": [3, 5], "x": 5, "y": 3},
+
+                {"matrix": [4, 2], "x": 4.5, "y": 4},
+                {"matrix": [4, 3], "x": 5.5, "y": 4},
+                {"matrix": [4, 4], "x": 6.5, "y": 4},
+                {"matrix": [4, 5], "x": 6.5, "y": 3},
+
+                {"matrix": [5, 0], "x": 14, "y": 0},
+                {"matrix": [5, 1], "x": 13, "y": 0},
+                {"matrix": [5, 2], "x": 12, "y": 0},
+                {"matrix": [5, 3], "x": 11, "y": 0},
+                {"matrix": [5, 4], "x": 10, "y": 0},
+                {"matrix": [5, 5], "x": 9,  "y": 0},
+
+                {"matrix": [6, 0], "x": 14, "y": 1},
+                {"matrix": [6, 1], "x": 13, "y": 1},
+                {"matrix": [6, 2], "x": 12, "y": 1},
+                {"matrix": [6, 3], "x": 11, "y": 1},
+                {"matrix": [6, 4], "x": 10, "y": 1},
+                {"matrix": [6, 5], "x": 9,  "y": 1},
+
+                {"matrix": [7, 0], "x": 14, "y": 2},
+                {"matrix": [7, 1], "x": 13, "y": 2},
+                {"matrix": [7, 2], "x": 12, "y": 2},
+                {"matrix": [7, 3], "x": 11, "y": 2},
+                {"matrix": [7, 4], "x": 10, "y": 2},
+                {"matrix": [7, 5], "x": 9,  "y": 2},
+
+                {"matrix": [8, 0], "x": 14, "y": 3},
+                {"matrix": [8, 1], "x": 13, "y": 3},
+                {"matrix": [8, 2], "x": 12, "y": 3},
+                {"matrix": [8, 3], "x": 11, "y": 3},
+                {"matrix": [8, 4], "x": 10, "y": 3},
+                {"matrix": [8, 5], "x": 9,  "y": 3},
+
+                {"matrix": [9, 2], "x": 10.5, "y": 4},
+                {"matrix": [9, 3], "x": 9.5, "y": 4},
+                {"matrix": [9, 4], "x": 8.5, "y": 4},
+                {"matrix": [9, 5], "x": 8.5, "y": 3}
+            ]
+          }
```

Comment:
matches `keymap.c`'s sequence/order of keycodes.
Accepting this suggestion will require, in `keymap.c`, moving inner most key's keycode in `keymap.c` to match this `layout` sequence. i.e. move inner most key's keycode from bottom row to 3rd row 
```suggestion
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},

                {"matrix": [5, 5], "x": 9,  "y": 0},
                {"matrix": [5, 4], "x": 10, "y": 0},
                {"matrix": [5, 3], "x": 11, "y": 0},
                {"matrix": [5, 2], "x": 12, "y": 0},
                {"matrix": [5, 1], "x": 13, "y": 0},
                {"matrix": [5, 0], "x": 14, "y": 0},

                {"matrix": [1, 0], "x": 0, "y": 1},
                {"matrix": [1, 1], "x": 1, "y": 1},
                {"matrix": [1, 2], "x": 2, "y": 1},
                {"matrix": [1, 3], "x": 3, "y": 1},
                {"matrix": [1, 4], "x": 4, "y": 1},
                {"matrix": [1, 5], "x": 5, "y": 1},

                {"matrix": [6, 5], "x": 9,  "y": 1},
                {"matrix": [6, 4], "x": 10, "y": 1},
                {"matrix": [6, 3], "x": 11, "y": 1},
                {"matrix": [6, 2], "x": 12, "y": 1},
                {"matrix": [6, 1], "x": 13, "y": 1},
                {"matrix": [6, 0], "x": 14, "y": 1},

                {"matrix": [2, 0], "x": 0, "y": 2},
                {"matrix": [2, 1], "x": 1, "y": 2},
                {"matrix": [2, 2], "x": 2, "y": 2},
                {"matrix": [2, 3], "x": 3, "y": 2},
                {"matrix": [2, 4], "x": 4, "y": 2},
                {"matrix": [2, 5], "x": 5, "y": 2},

                {"matrix": [7, 5], "x": 9,  "y": 2},
                {"matrix": [7, 4], "x": 10, "y": 2},
                {"matrix": [7, 3], "x": 11, "y": 2},
                {"matrix": [7, 2], "x": 12, "y": 2},
                {"matrix": [7, 1], "x": 13, "y": 2},
                {"matrix": [7, 0], "x": 14, "y": 2},

                {"matrix": [3, 0], "x": 0, "y": 3},
                {"matrix": [3, 1], "x": 1, "y": 3},
                {"matrix": [3, 2], "x": 2, "y": 3},
                {"matrix": [3, 3], "x": 3, "y": 3},
                {"matrix": [3, 4], "x": 4, "y": 3},
                {"matrix": [3, 5], "x": 5, "y": 3},
                {"matrix": [4, 5], "x": 6, "y": 2.5},

                {"matrix": [9, 5], "x": 8, "y": 2.5},
                {"matrix": [8, 5], "x": 9,  "y": 3},
                {"matrix": [8, 4], "x": 10, "y": 3},
                {"matrix": [8, 3], "x": 11, "y": 3},
                {"matrix": [8, 2], "x": 12, "y": 3},
                {"matrix": [8, 1], "x": 13, "y": 3},
                {"matrix": [8, 0], "x": 14, "y": 3},

                {"matrix": [4, 2], "x": 4.5, "y": 4},
                {"matrix": [4, 3], "x": 5.5, "y": 4},
                {"matrix": [4, 4], "x": 6.5, "y": 4},

                {"matrix": [9, 4], "x": 8.5, "y": 4},
                {"matrix": [9, 3], "x": 9.5, "y": 4},
                {"matrix": [9, 2], "x": 10.5, "y": 4}
            ]
        }
```